docs/_data/outreachy/outreachy_participants.json

```diff
@@ -17,6 +17,23 @@
         "github_handle": "bl-aire",
         "blog_url": "https://blaire.hashnode.dev/"
       }
+    ],
+    "community_members": [
+      {
+        "name": "Min Ragan-Kelley",
+        "github_handle": "minrk",
+        "roles": ["mentor"]
+      },
+      {
+        "name": "Georgiana Dolocon",
```

Comment:
```suggestion
        "name": "Georgiana Dolocan",
```

Reply:
So sorry @GeorgianaElena! I think I've fixed this now